index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Header Section (Toolbar)-->
+    <header class="header backgrnd-clr">
+      <nav class="navbar">
+        <img src="images/icons/Welcome.png" alt="logo" />
```

Comment:
The logo should be wrapped into an `<a></a>` tag.
### For Example
```html
<a>
  <img src="images/icons/Welcome.png" alt="logo" />
</a>
```
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.